src/common/components/CoreLayout/CoreLayout.tsx

```diff
@@ -1,3 +1,18 @@
+import { Navbar } from '@/common/components/Navbar/Navbar';
```

Comment:
```suggestion
import { Navbar } from '@/common/components/Navbar';
```

Can you add index.ts to the same folder?

Something like /components/Navbar/Navbar.tsx with the index.ts.

so `index.ts` will look something like this:

```
export * from "./Navbar";
```

 so that it is easier to maintain in the future!

Reply:
can we add index.ts to components folder instead?
so it will be like 
`import { Navbar } from '@/common/components`
i prefer not to keep creating index.ts
what do u think?

Reply:
sure. It would be great to stick to a better pattern here.
@Aztriltus thought on this?

Reply:
i would create index.ts for Icon components but the rest i will just add to index for components. I think this will reduce the imports line as well

Reply:
hmm. Do you think this will hamper searching File on vscode?

With `index.ts`
![CleanShot 2022-12-13 at 8 52 27](https://user-images.githubusercontent.com/53527664/207369722-375d2f42-ff45-4b07-96d7-a1b881bc89ca.png)

With Footer.tsx
![CleanShot 2022-12-13 at 8 52 57](https://user-images.githubusercontent.com/53527664/207369839-bd55c0ce-42f9-487a-9d04-478f60fe62fe.png)

Reply:
@sushanyadav i wasn't changing Footer.tsx to index.tsx, my suggestion was like this. But its okay, i can change to airfoil's preference
![image](https://user-images.githubusercontent.com/19740800/207378095-337b84a2-c46c-4450-99f2-6133f8ba04a8.png)

Reply:
I think with a single index in the parent folder, there may be minor conflicts to fix if multiple people work on a similar folder. 

Besides that, I think it doesn't have any issue with the ease of searching for the file or the import statement. 

I think we could give this a try and see how it goes?